userspace/libsinsp/gen_filter.cpp

```diff
@@ -431,6 +431,12 @@ std::string gen_event_filter_factory::filter_fieldclass_info::as_string(bool ver
 
 	for(auto &fld_info : fields)
 	{
+		// Skip fields with the EPF_TABLE_ONLY flag.
+		if(fld_info.tags.find("EPF_TABLE_ONLY") != fld_info.tags.end())
```

Comment:
Can this be refactored in a method, like:
`fld_info.is_skippable()`
?

Reply:
Good idea! Done.